examples/authorization/cmd/is_authorized.go

```diff
@@ -146,9 +146,47 @@ var withPropertyCmd = &cobra.Command{
 	},
 }
 
+var withExternalIDCmd = &cobra.Command{
+	Use:   "with_external_id",
+	Short: "Is Authorized by external id",
+	Long:  "Check if a digital twin is authorized to perform action based on external id",
+	Run: func(cmd *cobra.Command, args []string) {
+		var externalID authorizationpb.ExternalID
+		fmt.Print("Enter digital twin type: ")
+		fmt.Scanln(&(externalID.Type))
+		fmt.Print("Enter external id value: ")
+		fmt.Scanln(&(externalID.ExternalId))
+
+		resources := []*authorizationpb.IsAuthorizedRequest_Resource{
+			{
+				ExternalId: "resourceID",
+				Type:       "Type",
+				Actions:    []string{"ACTION"},
+			},
+		}
+		inputParams := map[string]*authorizationpb.InputParam{}
+		var policyTags []string
+
+		resp, err := client.IsAuthorizedByExternalID(
+			context.Background(),
+			&externalID,
+			resources,
+			inputParams,
+			policyTags,
+			retry.WithMax(2),
+		)
+
+		if err != nil {
+			log.Fatalf("failed to invoke operation on IndyKite Client %v", err)
+		}
```

Comment:
use `er` function or `RunE` instead of `Run`

Reply:
I've just copy&pasted another command in the file and don't know this stuff at all - would you point me anywhere so I can see what you mean?